### ISSUE TITLE
"__check_properties()" Method ("_lbh15.py") Improvement

### ISSUE THREAD

**Issue by lelaus** [newcleo-dev-team/lbh15#64]
- [1] rename the method as `__align_corrs_to_properties()`; it seems more aligned to what the method does;
- [2] the description is missing;
- [3] delete `prop_key` variable, since `key` is sufficient;
- [4] delete `remove_property` and `update_properties` boolean flags; their related if statements will be performed directly within the corresponding previous if statements if required;

within the first branch of the outer if statement, first branch of the inner if statement:
- [5] change the warning message into _Going to restore {property_name} from the {type(self).__name__}-related module, if any_; the current _default_ attribute could be misleading, since, for each liquid metal class, the variable `_default_corr_to_use` is specified too, while here the reference is to the property objects that are loaded from the modules set within the `_properties_modules_dict` variable;
- [6] add the property name to the list of the keys to remove from the correlation dict directly;
- [7] remove the call of the `correlations_available()` method when retrieving the correlations that are available for the current instance; the result of this call is already stored within the attribute `instance_available_corrs`, thus retrieving the available correlations only once when the instance is built;
- [8] within the `if key in corr_avail` statement, substitute the `update_properties` boolean flag with the direct call to the `__add_property()` method;

within the first branch of the outer if statement, second branch of the inner if statement:
- [9] the `remove_property` boolean flag should remain to `False`; the `__corr2use` dict is already updated; consequently, no key has to be inserted into the list of the correlation keys to remove;
- [10] substitute the `update_properties` boolean flag with the direct call to the `__add_property()` method;

within the second branch of the outer if statement:
- [11] the only thing to do, in addition to the warning message, is to update the correlation name by aligning it to the one within the `_properties`; then the statement `self.__corr2use[key] = self._default_corr_to_use[key]` should be changed in `self.__corr2use[key] = self.__properties[prop_key].correlation_name`;
- [12] the `remove_property"` should remain to `False` in any case, since the `__corr2use` dict is already updated, thus not requiring any item deletion;

- [13] once outside the outer for loop, the `keys_to_remove` list contains all the keys to remove from the `__corr2use` dict; by passing it to the _delete_corrs()_ private method, they are deleted from both the `__corr2use` and the `_correlations_to_use` dicts;

**Comment by panDanieleN** [comment]
Merged into master.

No need to update documentation and tests.
